src/main/resources/db/migration/V1_89__auth_user_validity_flag.sql

```diff
@@ -0,0 +1,3 @@
+alter table auth_user add column valid_hmpps_user boolean;
```

Comment:
i'm not sure we have used the term 'hmpps user' before - i'm not sure of its meaning. can we just keep the name simple like `is_valid` or `deleted` or something more generic? 

what do you think @sldblog ?

Reply:
should describe our intent with this flag, yeah -- `malformed` or `duplicate` etc

Reply:
extreme case could even be `selected_for_ticket123` 😁

Reply:
`duplicateOf uuid` ?

Reply:
I've changed the column to `deleted`. It defaults to `null`.